### PR TITLE
Change the Ukraine inbound rules for unvaxed travellers returning to England

### PR DESCRIPTION
**Pull request by gclssvglx**
### What
Reflect that un-vaccinated travellers from Ukraine no longer need to get tested when returning to England.

Change all `take a pre-departure COVID-19 test within the 3 days before you travel to England` instances to `2 days`.

### Why
The rules changed on 18 Feb.

### Travelling only to the Ukraine
|Before|After|
|:---|:---|
|Screenshot 2022-02-21 at 16 08 09|Screenshot 2022-02-21 at 16 07 35|

### Travelling to the Ukraine and one or more other countries
|Before|After|
|:---|:---|
|Screenshot 2022-02-21 at 16 05 51|Screenshot 2022-02-21 at 16 04 35|

Trello: https://trello.com/c/pGwvgYvT/693-change-ukraine-inbound-rules-for-unvaxed

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb [hunk 1]
```diff
@@ -13,11 +13,22 @@
   <p class="govuk-body">You must:</p>
 
   <ul class="govuk-list govuk-list--bullet">
-    <li><a href="https://www.gov.uk/guidance/coronavirus-covid-19-testing-for-people-travelling-to-england" class="govuk-link">take a pre-departure COVID-19 test</a> within the 3 days before you travel to England</li>
+    <% if calculator.travelling_to_ukraine? %>
+      <% if calculator.multiple_journey? %>
```
Comment (leenagupte) [inline app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb:17]: Ok, so if you are only travelling to Ukraine, the "pre-departure COVID-19 test" bullet isn't shown at all? Got it. It's a bit of head-scratcher isn't?

File: test/flows/check_travel_during_coronavirus_flow_test.rb [hunk 2]
```diff
@@ -609,5 +609,30 @@ class CheckTravelDuringCoronavirusFlowTest < ActiveSupport::TestCase
         assert_rendered_outcome text: "Returning to England if you’re not fully vaccinated"
       end
     end
+
+    context "content for Ukraine" do
```
Comment (leenagupte) [inline test/flows/check_travel_during_coronavirus_flow_test.rb:613]: Nice!

**Review by leenagupte — APPROVED** [review]
Looks good! The logic for not should one of the bullets if you're only travelling to Ukraine, made my brain hurt a little, but I can't think of a nicer way to do it and it works!

The way you've added the tests is great as they really clarify what's going on. ⭐

File: app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb (hunk 1 above)
Comment (gclssvglx) [reply app/flows/check_travel_during_coronavirus_flow/outcomes/_not_fully_vaxed.erb:17]: Yeah, sorry - couldn't find a better way 😢

**Pull request merged**